config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -104,6 +104,84 @@ presubmits:
         args:
         - runner.sh
         - ./scripts/ci-test.sh
+        env:
+        - name: KUBEBUILDER_ENVTEST_KUBERNETES_VERSION
+          value: "1.22.0"
+        resources:
+          requests:
+            cpu: 7300m
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api
+      testgrid-tab-name: capi-pr-test-main
+  - name: pull-cluster-api-test-main-k8s121
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    branches:
+    # The script this job runs is not in all branches.
+    - ^master$
+    - ^operator-0.4$
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210808-1eaeec7-1.21
+        args:
+        - runner.sh
+        - ./scripts/ci-test.sh
+        env:
+        - name: KUBEBUILDER_ENVTEST_KUBERNETES_VERSION
+          value: "1.21.2"
```

Comment:
is there anyway this could pull the latest 1.21.x version instead? so we don't have to update this spec everytime a patch comes out (or risk staying behind, this is already not the latest)

Reply:
We have somethings that can allow using version labels (e.g stable-1.21) in https://github.com/kubernetes-sigs/cluster-api/blob/b7aa6138aeeb8fd3094c3fe481f5af44576942e0/scripts/ci-e2e-lib.sh#L80

Reply:
this is how we do it in CAPZ https://github.com/chewong/cluster-api-provider-azure/blob/master/test/e2e/helpers.go#L565

Reply:
These are coming from kubebuilder, which are built ad-hoc https://github.com/kubernetes-sigs/kubebuilder/tree/tools-releases/build

Reply:
what would be the process to check if newer versions are available?

Reply:
I usually go in the repository to see if there is a newer version, I don't think there is any script to do so today

Reply:
could we document this as a comment so others know where this version is coming from?

Reply:
Sure thing!